eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -55,6 +55,7 @@ jobs:
 
     variables:
       - template: ../variables/globals.yml
+      jobSuffix: $[split(variables['Agent.JobName'], ' ')[1]]
```

Comment:
I don't think this syntax is going to work.

Reply:
right I am trying to figure out something that works.